azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/RuntimeSetting.java

```diff
@@ -69,6 +71,10 @@ public WebContainer getWebContainer() {
         return WebContainer.fromString(webContainer);
     }
 
+    public static String getLinuxJavaVersionByRuntimeStack(RuntimeStack runtimeStack) {
```

Comment:
Better naming. The returned string is not only Java version.

Reply:
yes, changed it to `getLinuxWebContainerByRuntimeStack`